### ISSUE TITLE
Add tooltip to Resource Library top-level elements

### ISSUE THREAD

**Issue by Azhrei** [RPTools/maptool#1156]
**Is your feature request related to a problem? Please describe.**
It's impossible to tell the directory that goes with a particular entry in the Resource Library.

**Describe the solution you'd like**
Add a tooltip that shows the directory path for top-level elements.

**Describe alternatives you've considered**
I know where Java Preferences are stored on Unix systems, so I know where to look for individual files that I can hand edit.  But Other users are less likely to know that, and Windows requires editing of the registry.

**Additional context**
Without this features, users can either (a) remove entries they aren't sure about and then manually add them back in again, or...  Well, there are no other options.

The tooltip should convert the home directory (the `user.home` System property) into `~/` for compatibility with various documentation.

**Comment by Phergus** [comment]
Huh?  You mean like what directory Default is in?

**Comment by Azhrei** [comment]
Yep.  I had three entries that all were named **Default** and I couldn't tell which one I needed to remove. 🤔

**Comment by Phergus** [comment]
Remove them all using a current version of MapTool and then Help -> Add Default images.

Not against adding the tooltip.

**Comment by Azhrei** [comment]
Yeah, that’s what I did.  I looked at editing the Preferences directly, but even there I couldn’t tell which was which, as the Prefs just saved a list of paths...

**Comment by Azhrei** [comment]
Another issue related to this one — the preview images also don't include any information about the path used to find them.

This means when I want to change the token image, I can find the replacement in the Resource Library using the search, but I don't know where it is to select it in the token editor?!

It would be nice if it could be dragged from the Resource Library or copy/pasted into the token editor, but this issue is just about adding a tooltip to identify the directories in the resource library, so adding a tooltip to preview images seems in the same ballpark.
